packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -130,6 +130,14 @@ export default function BookmarkFeedLayout({
     };
   }, [searchQuery, feedQueryKey, listId, isReminderOnly, isFolderPage]);
 
+  useEffect(() => {
+    setIsHydrated(true);
+  }, []);
+
+  if (!isHydrated) {
+    return null;
+  }
```

Comment:
Since it is coming from `useViewSize` inside `useFeedLayout`, you can try using `useViewSizeClient`, it is the same hook only difference is that on the server it will always return `false` for server and first render so there should be hydration issue.

Reply:
Cool. Let me look into that.

Reply:
Yeah, it worked as intended.

Reply:
Since `useFeedLayout` is used in different pages, maybe just check it does not introduce layout shifts on those pages as well, but it should be ok if this worked.